packages/settingeditor/src/inspector.ts

```diff
@@ -76,7 +78,10 @@ class InspectorConnector extends DataConnector<
         const errors = this._validate(request.text);
 
         if (!errors) {
-          return resolve(null);
+          return resolve({
+            data: { 'text/markdown': 'No errors found' },
```

Comment:
Does `text/plain` work here? Seems like it would be slightly less processing if we don't claim it is markdown.

Reply:
@jasongrout I tried `text/plain` but it caused vertical alignment issues